dpe/src/dpe_instance.rs

```diff
@@ -73,7 +73,16 @@ impl DpeInstance<'_> {
         env: &mut impl DpeEnv,
         support: Support,
     ) -> Result<DpeInstance<'a>, DpeErrorCode> {
-        const TEST_ISSUER: &[u8] = b"Test Issuer";
+        // DER encoded issuer with CN
```

Comment:
Can you add a comment for how you generated this? Also the mix of decimal numbers and ascii bytes is a bit odd.